test/cljs/status_im/test/models/network.cljs

```diff
@@ -0,0 +1,74 @@
+(ns status-im.test.models.network
+  (:require [cljs.test :refer-macros [deftest is testing]]
+            [status-im.models.network :as model]))
+
+(deftest valid-rpc-url-test
+  (testing "nil?"
+    (is (not (model/valid-rpc-url? nil))))
+  (testing "a blank url"
+    (is (not (model/valid-rpc-url? ""))))
+  (testing "a url without a protocol"
+    (is (not (model/valid-rpc-url? "something"))))
+  (testing "a url without a hostname"
+    (is (not (model/valid-rpc-url? "https://"))))
+  (testing "an http url"
+    (is (model/valid-rpc-url? "http://localhost")))
+  (testing "an https url"
+    (is (model/valid-rpc-url? "https://localhost.something.else"))))
```

Comment:
Specifying `port` is not required? Does it use default port like `8545`?

Reply:
@dmitryn good point, our use only the default port, but I will add the port to the validation

Reply:
Actually it's loose enough the regex that it validates ok with a port, but added a test case.